### PR TITLE
fix: reword comparison report doc

### PR DESCRIPTION
**Pull request by alexbarros**

### CODE REVIEW

File: docsrc/source/pages/use_cases/comparing_datasets.rst [hunk 1]
```diff
@@ -42,11 +40,9 @@ In order to compare more than two reports, the following syntax can be used:
     # Save report to file
     comparison_report.to_file("comparison.html")
 
-Note that generating reports for three or more datasets is not (yet) fully supported.
+Note that this functionality only ensures the support to generate reports to compare two datasets.
```
Comment (akx) [inline docsrc/source/pages/use_cases/comparing_datasets.rst:43]: I think the proposed wording is harder to read than the original.

File: docsrc/source/pages/use_cases/comparing_datasets.rst [hunk 2]
```diff
@@ -42,11 +40,9 @@ In order to compare more than two reports, the following syntax can be used:
...
 It is possible to obtain the statistics - the report may have formatting issues.
 One of the settings that can be changed to improve the formatting is ``settings.report.precision``.
-As a rule of thumb the value 10 can be used for a single report, 8 for comparing two reports and 5 for comparing three reports.
-
-.. pull-quote::
-
-    ⌛ Interested in uncovering more temporal patterns? Check out `popmon <https://github.com/ing-bank/popmon>`_.
+In case your report has formatting issues, you can improve it by tweaking the settings.
+One of the settings that can be changed is settings.report.precision.
+As a rule of thumb, the value 10 can be used for a single report and 8 for comparing two reports.
```
Comment (akx) [inline docsrc/source/pages/use_cases/comparing_datasets.rst:48]: This seems to be duplicating the three lines above.

File: docsrc/source/pages/use_cases/comparing_datasets.rst [hunk 3]
```diff
@@ -2,8 +2,6 @@
 Dataset Comparison
 ==================
 
-*This feature was introduced in pandas-profiling 3.4.*
```
Comment (akx) [inline docsrc/source/pages/use_cases/comparing_datasets.rst:5]: Why would this be removed? Python's docs note which version adds each new function, and it's easy to accidentally be reading docs for a version more recent than what you're using.

File: docsrc/source/pages/use_cases/comparing_datasets.rst [hunk 4]
```diff
@@ -42,11 +42,7 @@ In order to compare more than two reports, the following syntax can be used:
...
-Note that generating reports for three or more datasets is not (yet) fully supported.
+Note that generating comparison reports for three or more datasets is not (yet) supported.
 It is possible to obtain the statistics - the report may have formatting issues.
-One of the settings that can be changed to improve the formatting is ``settings.report.precision``.
-As a rule of thumb the value 10 can be used for a single report, 8 for comparing two reports and 5 for comparing three reports.
-
-.. pull-quote::
-
-    ⌛ Interested in uncovering more temporal patterns? Check out `popmon <https://github.com/ing-bank/popmon>`_.
+One of the settings that can be changed is settings.report.precision.
```
Comment (akx) [inline docsrc/source/pages/use_cases/comparing_datasets.rst:47]: The code fragment should remain as such.
```suggestion
One of the settings that can be changed is ``settings.report.precision``.
```

File: docsrc/source/pages/use_cases/comparing_datasets.rst (hunk 1 above)
Comment (fabclmnt) [reply docsrc/source/pages/use_cases/comparing_datasets.rst]: We won't include the (yet) fully supported. Hence the change.

File: docsrc/source/pages/use_cases/comparing_datasets.rst (hunk 3 above)
Comment (fabclmnt) [reply docsrc/source/pages/use_cases/comparing_datasets.rst]: This is not the correct version of the feature. Rest assured that by the time the release process version information will be included in the docs.

**Pull request merged**